custom_components/mikrotik_router/coordinator.py

```diff
@@ -516,11 +517,28 @@ def get_capabilities(self):
             self.support_ppp = True
             self.support_wireless = True
```

Comment:
problem comes from here, 

I guess the solution could be to remove assigning True from here and add it in every branch as is, except the last one, in last one we have to check whether we on 7.13 and newer or not:

```python
        elif 0 < self.major_fw_version >= 7:
            self.support_ppp = True
            if "wifiwave2" in packages and packages["wifiwave2"]["enabled"]:
                self.support_capsman = False
                self.support_wireless = True
                self._wifimodule = "wifiwave2"

            elif "wifi" in packages and packages["wifi"]["enabled"]:
                self.support_capsman = False
                self.support_wireless = True
                self._wifimodule = "wifi"

            elif "wifi-qcom" in packages and packages["wifi-qcom"]["enabled"]:
                self.support_capsman = False
                self.support_wireless = True
                self._wifimodule = "wifi"

            elif "wifi-qcom-ac" in packages and packages["wifi-qcom-ac"]["enabled"]:
                self.support_capsman = False
                self.support_wireless = True
                self._wifimodule = "wifi"

            else:
                self.support_capsman = True
                self.support_wireless = self.major_fw_version >= 7 and self.minor_fw_version < 13
```

Reply:
At least this solution works fine for my setup

Reply:
Hello, I've updated the PR to include this. I have also tested this code on a CSR309-1G-8S+ on 7.13, hAP ac3 on 7.13 and CRS328-24P-4S+ on 7.13. I didn't test on < 7.13